SECURITY.md

```diff
@@ -0,0 +1,50 @@
+## Security Policy
+
+The Wikijump Team prioritizes security and will fix issues as soon as they are disclosed.
+
+If you discover a security vulnerability, please privately contact the Wikijump Security Team. This can be done by emailing security@wikijump.org,
+or reaching out to a member individually:
+
+| GitHub                                         | Wikidot                                                                | Discord           | Email                     |
+|------------------------------------------------|------------------------------------------------------------------------|-------------------|---------------------------|
+| [@ammongit](https://github.com/ammongit)       | [aismallard](https://www.wikidot.com/account/messages#/new/4598089)    | `aismallard#0002` | ammon.i.smith@gmail.com   |
+| [@stormbreath](https://github.com/stormbreath) | [stormbreath](https://www.wikidot.com/account/messages#/new/3075960)   |                   |                           |
+| [@rossjrw](https://github.com/rossjrw)         | [Croquembouche](https://www.wikidot.com/account/messages#/new/2893766) |                   | ross@rossjrw.com          |
+| [@danieltharp](https://github.com/danieltharp) | [pxdnbluesoul](https://www.wikidot.com/account/messages#/new/1414125)  |                   | daniel.tharp@gmail.com    |
+
+### Wikijump
+
+After receiving a report of a security issue within Wikijump, we will validate and determine its scope.
```

Comment:
I'd like to see an SLA for the initial response as well. For example, Rust has:

> Your email will be acknowledged within 24 hours, and you’ll receive a more detailed response to your email within 48 hours indicating the next steps in handling your report.

https://www.rust-lang.org/policies/security

Reply:
Not sure. My thinking was that it's good to have _some_ time frame there, to set expectations on when we'll respond. But I don't have an opinion on what to set it to.